jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1073,6 +1079,22 @@ private HttpFields takeTrailers()
             return trailers;
         }
 
+        @Override
+        public void write(Content.Chunk chunk, Callback callback)
```

Comment:
might need to define this method in the interface so we can add javadoc saying that Trailers are handled (and what steps have to be done to get them)